### PR TITLE
Update Introduction.md

### PR DESCRIPTION
**Pull request by zaucker**
Fix typos and grammar.

### CODE REVIEW

File: docs/Introduction.md [hunk 1]
```diff
@@ -818,7 +818,7 @@ const validate = ctx.schema('testForm');
 
 ### Home
 
-The directory in which the main application script resides, usually called `index.js` is considered the application
+The directory in which the main application script, usually called `index.js`, resides, is considered the application
```
Comment (kraih) [inline docs/Introduction.md:821]: This looks weird to me.

Comment (zaucker) [reply docs/Introduction.md:821]: At a minimum I think there should be a comma after `index.js` in the original version.
Feel free to remove or change this, I have allowed edits by maintainers.

Comment (kraih) [reply docs/Introduction.md:821]: Yes, just add the comma after `index.js`, and leave the sentence as is otherwise.

Comment (zaucker) [reply docs/Introduction.md]: Done.

**Comment by kraih** [comment]
Now just squash the commits please.

**Comment by zaucker** [comment]
> Now just squash the commits please.

Don't know how to do this from the GitHub GUI. I think you should be able to squash on merging.

**Comment by kraih** [comment]
Don't know if GitHub supports it, i usually just use `git rebase -i ...`. Also bonus points for a better commit message, like `Fix a few typos in Introduction.md` or so.

**Comment by zaucker** [comment]
Will make a new PR

File: docs/Introduction.md (hunk 1 above)
Comment (Grinnz) [reply docs/Introduction.md]: How about:

The directory in which the main application script (usually called `index.js`) resides is considered the application home directory.

Comment (kraih) [reply docs/Introduction.md]: That would have worked too.

**Comment by jhthorsen** [comment]
@zaucker: You can also rebase locally and force push to the same branch.

**Comment by zaucker** [comment]
> @zaucker: You can also rebase locally and force push to the same branch.

Yes, but I didn't have a local checkout an d just edited the file on GitHub.

**Comment by zaucker** [comment]
Sorry for having tried to make a small contribution. Will refrain from doing so in the future.

**Comment by jhthorsen** [comment]
@zaucker: I think you misunderstood our feedback: It was meant as help and guidance, since many people don't know these things.

**Comment by kraih** [comment]
Yes, the feedback was meant to help you... but this has also been a reminder for me to follow our own rules more strictly again, because they exist for a reason. I should have rejected this pull request, instead of letting it lead to this misunderstanding, which left everyone involved being unhappy.

> Please don't send pull requests for overly simplistic changes, such as the addition of a comma or semicolon.

https://docs.mojolicious.org/Mojolicious/Guides/Contributing#CONTRIBUTING-DOCUMENTATION

**Comment by kraih** [comment]
> Ok, you don't want PRs to fix typos or grammar. Fair enough, it didn't occur to me to check the contribution guidelines for such a rule.

Not quite so simple. It's about simplistic pull requests. If someone was to do a full typo check on all the docs, that would be perfectly fine. But ten separate pull requests to fix individual typos would cause more review work than the fixes would be worth.

Which brings me back to this PR. Time is the most precious resource this project has, and two members of the team spent a not insignificant amount of it to make sure your PR would be successful and to help you be more efficient in the future. An antagonistic reaction to that is quite unfair.

**Pull request closed without merging**